docs/core/deploy.md

```diff
@@ -34,11 +41,13 @@ To be able to use the autocomplete feature of the point inputs you have to:
 
 ## Worldwide Setup
 
-GraphHopper is able to handle coverage for the whole [OpenStreetMap road network](http://planet.osm.org/).
-It needs approximately 25GB RAM for the import (CAR only) and ~1 hour (plus ~5h for contraction).
-If you can accept slower import times this can be reduced to 14GB RAM - you'll need to set `datareader.dataaccess=MMAP` in the config file.
+GraphHopper is able to handle coverage for the world-wide [OpenStreetMap road network](http://planet.osm.org/).
```

Comment:
GraphHopper can handle the world-wide OpenStreetMap road network

Reply:
Maybe mention different map sizes, for example planet car has around 415mio edges, Europe 150mio, North America 86mio.

Reply:
Maybe mention different map sizes, for example planet car has around 415mio edges, Europe 150mio, North America 86mio.